ui/flow.reel/flow.js

```diff
@@ -1290,6 +1290,11 @@ var Flow = exports.Flow = Component.specialize( /** @lends Flow.prototype # */ {
                 j,
                 i;
 
+            if (this._needsClearVisibleIndexes) {
+                this._visibleIndexes.splice(newVisibleIndexes.length, Infinity);
```

Comment:
Are you sure about this splice(), the use of Infinity vs doing this._visibleIndexes.length - newVisibleIndexes.length ?